Packages/doc/databrowser_paplot.rst

```diff
@@ -0,0 +1,223 @@
+.. _db_paplot:
+
+Pulse Average (PA) plot
+=======================
+
+When the stimulus set contains multiple pulses the pulse average plot allows to
+extract these single pulses to overlay, and average and visually inspect these.
+
+Although the stimulus set will contain :ref:`wavebuilder_pulsetrain` epochs in
+most cases, the pulse extraction algorithm works with all epoch types and
+special measurement modes like :ref:`daephys_dDAQ` or :ref:`daephys_oodDAQ`.
+
+The dDAQ slider from the :ref:`db_settings` tab is respected as is the :ref:`db_channelselection`.
+
+For instructional purposes assume you have acquired a pulse train with oodDAQ on 4 different headstages.
+This means you have 4 active headstages with each a DA and AD channel. We
+only visualize the AD data in the PA plot.
+
+The PA plot will then have 16 = 4 x 4 PA sets (either in one big graph or in multiple graphs).
+
+Each set will have one source of pulse starting times (called "region")
+and one source of the visualized data. The pulse starting times are always
+extracted from the DA channel of the region.
```

Comment:
To gather PA sets for a column of (4) sets, pulse timing data from a DA channel (of a headstage) are used time-align AD regions for each active headstage.